### PR TITLE
ci: refactor test reporter

### PR DESCRIPTION
**Pull request by jerensl**
**Description**

https://github.com/actions/github-script?tab=readme-ov-file#reading-step-results

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/extension-test-on-pr.yml [hunk 1]
```diff
@@ -263,23 +263,29 @@ jobs:
...
           REMOTE_PROVIDER_USER_EMAIL: ${{ secrets.LAYER5_CLOUD_TESTING_BOT_EMAIL }}
           REMOTE_PROVIDER_USER_PASSWORD: ${{ secrets.LAYER5_CLOUD_TESTING_BOT_PASSWORD }}
-
+      - name: Test reporter
+        uses: actions/github-script@v7
+        id: test-reports
+        with:
+          script: |
+              const fs = require('fs')
+              const E2ETestReporter = String(fs.readFileSync('./meshmap/test-report.md'))
```
Comment (leecalcote) [inline .github/workflows/extension-test-on-pr.yml:279]: Will you change the filename? We are moving away from the name @meshmap”.”

Comment (jerensl) [reply .github/workflows/extension-test-on-pr.yml:279]: It's the name of the folder in meshery extension repo, @aabidsofi19 what do u think? it's safe now to change the folder name?

Comment (aabidsofi19) [reply .github/workflows/extension-test-on-pr.yml:279]: yeah , it would require refactoring across worflows and build scripts.

**Review by aabidsofi19 — APPROVED** [review]

File: .github/workflows/extension-test-on-pr.yml (hunk 1 above)
Comment (aabidsofi19) [inline .github/workflows/extension-test-on-pr.yml:279]: why are we not adding the custom reporter in playwright config

**Review by aabidsofi19 — CHANGES_REQUESTED** [review]

File: .github/workflows/extension-test-on-pr.yml (hunk 1 above)
Comment (jerensl) [reply .github/workflows/extension-test-on-pr.yml:279]: @aabidsofi19 The custom report has already been added to the Meshery extension repo, this file is the result of the reporting which is separated from logs

**Pull request merged**